arm-sql/2014-04-01/swagger/geoBackupPolicies.json

```diff
@@ -0,0 +1,307 @@
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database."
+          },
+          {
+            "name": "geoBackupPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the geo backup policy."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeoBackupPolicy"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/geoBackupPolicies": {
+      "get": {
+        "tags": [
+          "GeoBackupPolicies"
+        ],
+        "operationId": "GeoBackupPolicies_ListByDatabase",
+        "description": "Returns a list of geo backup policies.",
+        "x-ms-examples": {
+          "List geo backup policies": { "$ref": "../examples/GeoBackupPoliciesList.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeoBackupPolicyListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "GeoBackupPolicyProperties": {
+      "properties": {
+        "state": {
+          "type": "string",
+          "description": "The state of the geo backup policy."
+        },
+        "storageType": {
+          "type": "string",
+          "description": "The storage type of the geo backup policy.",
+          "readOnly": true
+        }
+      },
+      "required": ["state"],
+      "description": "The properties of the geo backup policy."
+    },
+    "GeoBackupPolicy": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GeoBackupPolicyProperties",
+          "description": "The properties of the geo backup policy.",
+          "x-ms-client-flatten": true
+        },
+        "kind": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Kind of geo backup policy.  This is metadata used for the Azure portal experience."
+        },
+        "location":{
```

Comment:
Mutability for tracked resources is [create, read] but since this resource can't actually be created, I have it as readOnly.

Do tracked resources always have tags?  This resource didn't have them in fiddler.